include/debug.h

```diff
@@ -34,6 +34,10 @@
 #ifdef LMMS_DEBUG
 	#include <assert.h>
 #else
+	// TODO: assert is a standard macro, we should choose another name
```

Comment:
>Isn't this quite common practice though?

Using `assert` for operations with side effects? Yes, it is a common error. We should not change the meaning of standard `assert`.